OxylusEditor/Resources/Shaders/scene.slang

```diff
@@ -7,6 +7,12 @@ public const static f32 CAMERA_SCALE_UNIT     = 0.01;
 public const static f32 INV_CAMERA_SCALE_UNIT = 1.0 / CAMERA_SCALE_UNIT;
 public const static f32 PLANET_RADIUS_OFFSET  = 0.001;
 
+public enum class PassConfig : u32 {
+  None = 0,
+  EnableBloom = 1 << 0,
+  EnableFXAA = 1 << 1,
+};
```

Comment:
slang has special attribute for enums called `[Flags]` and it will automatically assign bit flags to it, but since c++ doesnt have such feature it might be invisible when doing fuzzy search, etc
it's up to you